bundles/org.openhab.binding.sonyprojector/src/main/java/org/openhab/binding/sonyprojector/internal/communication/SonyProjectorConnector.java

```diff
@@ -971,40 +969,37 @@ private synchronized void sendIR(SonyProjectorItem item) throws SonyProjectorExc
      *
      * @return the buffer containing the returned message
      *
-     * @throws SonyProjectorException - In case of any problem
+     * @throws ConnectionException - In case of any connection problem
+     * @throws CommunicationException - In case of any communication problem
```

Comment:
```suggestion
     * @throws ConnectionException In case of any connection problem
     * @throws CommunicationException In case of any communication problem
```